microsoft/testsuites/core/serial_console.py

```diff
@@ -0,0 +1,38 @@
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.features import SerialConsole
+from lisa.sut_orchestrator.azure.features import SerialConsole as AzureSerialConsole
```

Comment:
The method should be defined in general serialconsole too, it's not an Azure only feature.